### PR TITLE
Add view for 'no projects yet' -page

### PR DESCRIPTION
**Pull request by PatricKangasmaki**
There's no users without roadmaps currently so to test this, you need to remove someone from their roadmaps and then log in. Create a new project button doesn't do anything yet, it needs to be updated later when the modal is ready.

### CODE REVIEW

File: frontend/src/pages/CreateProjectPage.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <ModalHeader>{t('Greet user', { userInfo })}</ModalHeader>
+        <ModalContent>
+          <div className={classes(css.about)}>
+            <Trans i18nKey="No projects yet" />
+          </div>
+          <div className={classes(css.body)}>
+            <Trans i18nKey="First project info 1/2" />
+          </div>
+          <div className={classes(css.body)}>
+            <Trans i18nKey="First project info 1/2" />
```
Comment (IidaKainu) [inline frontend/src/pages/CreateProjectPage.tsx:38]: Looks like a typo: the first info is repeated twice
```suggestion
            <Trans i18nKey="First project info 2/2" />
```

**Review by IidaKainu — CHANGES_REQUESTED** [review]
Looks good!
There's some similar css in create project, login and register pages (white container/formDiv, footer). They could be combined 👍

**Comment by PatricKangasmaki** [comment]
Fixed the typo and moved css-classes that were used in multiple places to the shared.scss folder 👍 I left the footer in the createProject-page module untouched since it has its own translations and positions

File: frontend/src/components/LoginNavBar.tsx [hunk 2]
```diff
@@ -10,26 +10,37 @@ import css from './LoginNavBar.module.scss';
...
         <Trans i18nKey="Register" />
       </Link>
-    ) : (
+    )}
+    {type === 'landing' && (
+      <Link className={classes(css.loginButton)} to={paths.loginPage}>
+        <Trans i18nKey="Login" />
+      </Link>
+    )}
+    {type === 'register' && (
```
Comment (juhoeerola) [inline frontend/src/components/LoginNavBar.tsx:43]: These could maybe be combined
```suggestion
    {(type === 'register' || type === 'landing') && (
```

File: frontend/src/components/NavBar.tsx [hunk 3]
```diff
@@ -31,26 +31,29 @@ export const NavBar = () => {
...
-              <RoadmapSelectorWidget />
-              <div className={classes(css.navBarDivider)} />
-              <Link className={classes(css.navBarLink)} to={paths.userInfo}>
-                <PermIdentityIcon className={classes(css.icon)} />
-              </Link>
-              <Link className={classes(css.navBarLink)} to={paths.logoutPage}>
-                <PowerSettingsNewIcon className={classes(css.icon)} />
-              </Link>
-            </>
+      {userInfo &&
```
Comment (juhoeerola) [inline frontend/src/components/NavBar.tsx:34]: I think this whole component could be refactored a bit

Using couple conditional early returns would make this much clearer in my opinion, e.g.

```ts
  if (!userInfo) {
    return pathname.startsWith(paths.loginPage) ? (
      <LoginNavBar type="login" />
    ) : (
      <LoginNavBar type="register" />
    );
  }

  if (pathname.startsWith(paths.getStarted))
    return <LoginNavBar type="getStarted" />;

  return (
    <div className={classes(css.navBarDiv)}>
      <CornerPiece />
      <div className={classes(css.navBarRightSide)}>
        <div className={classes(css.navBarText)}>
          <Trans i18nKey="Project" />
        </div>
        <RoadmapSelectorWidget />
        <div className={classes(css.navBarDivider)} />
        <Link className={classes(css.navBarLink)} to={paths.userInfo}>
          <PermIdentityIcon className={classes(css.icon)} />
        </Link>
        <Link className={classes(css.navBarLink)} to={paths.logoutPage}>
          <PowerSettingsNewIcon className={classes(css.icon)} />
        </Link>
      </div>
    </div>
  );
```

**Review by juhoeerola — CHANGES_REQUESTED** [review]

**Comment by PatricKangasmaki** [comment]
Added all your suggestions!

**Review by juhoeerola — APPROVED** [review]
Look good 👍 

Which ever is merged later, this or #116, can add the functionality to the "Create new project" button

**Review by IidaKainu — APPROVED** [review]
👍

**Pull request merged**